tests/network/connectivity/test_pod_network.py

```diff
@@ -61,12 +62,18 @@ def pod_net_vmb(
 
 @pytest.fixture()
 def pod_net_running_vma(pod_net_vma):
-    return running_vm(vm=pod_net_vma, wait_for_cloud_init=True)
+    pod_net_vma.vmi.wait_for_condition(
```

Comment:
@EdDev Please note that such change can be applied for cases like this, where the VM was already started in the creation fixture (`pod_net_vma` includes a call to `start()`).
For other cases, where the VM is currently started by the call to `running_vm`, we will have to add the start operation (maybe by changing these flows and create the VMs by calling `create_running_vm()`?).
Other than that - LGTM.